### ISSUE TITLE
Enter a short summary here!

### ISSUE THREAD

**Issue by iCyberXoji** [jeffhill/Dota2#325]
### Ability name

Astral step

### Description

Yesterday during the match, my teamate who choose to play on Tauren, used Astral Step skill and moved it to enemy hero Invoker, Invoker blinked to him and gave a hex, astral step returned, bkb turned on but he remained a chicken but with bkb. I forgot the timestamp Sorry for the inconvenience, i will set randomly 15 min.

### Example Match ID (and possibly Timestamp)

6598742763 - 15:00

**Comment by IshaanGakhar** [comment]
Hex is undispellable. Even LC's W, lotus, abbadon ult (Hard Dispel) cannot dispel hex. It is working as intended.

**Comment by Phllips** [comment]
> Hex is undispellable. Even LC's W, lotus, abbadon ult (Hard Dispel) cannot dispel hex. It is working as intended.
This is wrong, hex is dispellable with hard dispel, however this is working as intended as astral spirit only gives a BASIC DISPEL not a hard dispel

**Comment by CYAXARES-76** [comment]
Only STRONG Dispel can debuff the Hex. Astral Spirit Agh was only BASIC Dispel.

**Comment by shaileshgarg** [comment]
This can be closed.
